tests/travis-integration.sh

```diff
@@ -6,7 +6,7 @@ set -o errexit
 
 source .tox/$TOXENV/bin/activate
 
-until curl http://boulder:4000/directory 2>/dev/null; do
+until curl http://localhost:4000/directory 2>/dev/null; do
```

Comment:
I'd like to see this change removed. Per the name of this script, it was written to run in Travis where the informative name `boulder` works properly. To run integration tests locally, you should follow the instructions found at [Integration testing with the Boulder CA](https://certbot.eff.org/docs/contributing.html#integration-testing-with-the-boulder-ca).